packages/terra-list/src/ListSubsectionHeader.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ListUtils from './ListUtils';
+import styles from './ListSubsectionHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Whether or not the subsection is collapsed.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Whether or not the subsection can be collapsed.
+   */
+  isCollapsible: PropTypes.bool,
+  /**
+   * Optionally sets the heading level. One of `2`, `3`, `4`, `5`, `6`.
+   */
+  level: PropTypes.oneOf([2, 3, 4, 5, 6]),
+  /**
+   * The associated metaData to be provided in the onSelect callback.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  metaData: PropTypes.object,
+  /**
+   * Function callback for when the appropriate click or key action is performed.
+   * Callback contains the javascript evnt and prop metadata, e.g. onSelect(event, metaData)
+   */
+  onSelect: PropTypes.func,
+  /**
+   * Function callback passthrough for the ref of the section li.
+   */
+  refCallback: PropTypes.func,
+  /**
+   * Title text to be placed within the subsection header.
+   */
+  title: PropTypes.string,
+};
+
+const defaultProps = {
+  isCollapsed: false,
+  isCollapsible: false,
+  level: 2,
+  title: '',
+};
+
+const SectionHeader = ({
+  isCollapsed,
+  isCollapsible,
+  level,
+  metaData,
+  onBlur,
+  onClick,
+  onKeyDown,
+  onMouseDown,
+  onSelect,
+  refCallback,
+  title,
+  ...customProps
+}) => {
+  const sectionHeaderClassNames = cx([
+    'subsection-header',
+    { 'is-collapsible': isCollapsible },
+    customProps.className,
+  ]);
+
+  const ariaSpread = {};
```

Comment:
nit: I'd go with `attrSpread` for this since its changing more than just aria attributes.

Reply:
Updated.